specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -524,6 +629,86 @@
           "minimum": 1,
           "maximum": 102400,
           "description": "The maximum size of the share, in gigabytes. Must be greater than 0, and less than or equal to 5TB (5120). For Large File Shares, the maximum size is 102400."
+        },
+        "enabledProtocols": {
+          "type": "string",
+          "enum": [
+            "SMB",
+            "NFS"
```

Comment:
As we get on the last week, NFS feature has server issue, and seems won't be ready before June. (correct me if not)
@zfchen95  Would you please check with Will that he really want to open NFS in swagger now? If the change is in swagger, means it's open in rest API and all SDKs. Then customer might use it with SDK, and if feature is not ready, we might will get many customer report issues.

Reply:
I forwarded you an email from Ran. We still plan to public preview the feature in limited regions.

Reply:
See, thanks!